client/src/modules/chapters/pages/chaptersPage.tsx

```diff
@@ -24,6 +25,13 @@ export const ChaptersPage: NextPage = () => {
           </LinkButton>
         )}
       </Flex>
+      <Link
+        href={`/events/`}
+        _hover={{ color: 'blue' }}
+        textDecoration={'underline'}
+      >
+        View events list
+      </Link>
```

Comment:
I'm wondering what user journey this helps enable.  I totally get the desire to have links from the home page, but if a user has arrived on the chapters page, I would assume they're just interested in seeing the chapters.